src/parkapi_sources/converters/opendata_swiss/models.py

```diff
@@ -0,0 +1,200 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from collections import defaultdict
+from datetime import datetime, timezone
+from enum import Enum
+from typing import Optional
+
+from validataclass.dataclasses import validataclass
+from validataclass.exceptions import ValidationError
+from validataclass.validators import DataclassValidator, EnumValidator, IntegerValidator, ListValidator, Noneable, StringValidator
+
+from parkapi_sources.converters.base_converter.pull import GeojsonFeatureGeometryInput
+from parkapi_sources.models import RealtimeParkingSiteInput, StaticParkingSiteInput
+from parkapi_sources.models.enums import ParkAndRideType, ParkingSiteType, PurposeType
+
+
+@validataclass
+class OpenDataSwissAddressInput:
```

Comment:
Are you sure about all the `Noneable`s in the following `validataclasses`? If I look at the data, most of it is well set and not `null`. For example, there is not a single `addressLine` which is `null`. Please have a look at the data and use `Noneable` just when it's needed.